components/VM_Arm/configurations/vm.h

```diff
@@ -68,22 +68,28 @@
     attribute int num_extra_frame_caps; \
     attribute int extra_frame_map_address; \
     attribute { \
-        string linux_ram_base; \
-        string linux_ram_paddr_base; \
-        string linux_ram_size; \
-        string linux_ram_offset; \
+        string ram_base; \
+        string ram_paddr_base; \
+        string ram_size; \
+        string ram_offset; \
         string dtb_addr; \
         string initrd_max_size; \
         string initrd_addr; \
-    } linux_address_config; \
+        string kernel_entry_addr = "-1"; \
```

Comment:
We can use `ENTRY_ADDR_NOT_SET`here:
```suggestion
        string kernel_entry_addr = VAR_STRINGIZE(ENTRY_ADDR_NOT_SET); \
```

Reply:
When I do that, I get:

```
kernel_entry_addr = "ENTRY_ADDR_NOT_SET",
```

`strtoul` defaults to zero, so  the logic wouldn't apply, since technically a guest could have an entry address of zero. We'd have to pass that flag through via the `cpp_flags`

```
set(cpp_flags "-DKERNELARMPLATFORM_EXYNOS5410")
```

Reply:
Hmm.... I though the double macro layers in `VAR_STRINGIZE` are supposed to solve exactly this. Then let's remove the define `ENTRY_ADDR_NOT_SET` and use `-1` in both cases. We can clean this up later then

Reply:
That's how I currently have it. `ENTRY_ADDR_NOT_SET` is defined as `-1`, which makes the C code more readable.

Reply:
Fair point, let's keep it then.